src/types/ListRenderer.ts

```diff
@@ -0,0 +1,7 @@
+import { CheckListRenderer, OrderedListRenderer, UnorderedListRenderer } from '../ListRenderer';
+
+/**
+ * @todo move to types
+ * Type that represents all possible list renderer types
+ */
+export type ListRendererTypes = OrderedListRenderer | UnorderedListRenderer | CheckListRenderer;
```

Comment:
could not be renamed like this because `ListRenderer` is an abstract class that is extended by each of the `OrderedListRenderer | UnorderedListRenderer | CheckListRenderer`

Reply:
you can rename abstract class to ListRendererBase

Reply:
sorry, we have interface, i renamed and solved issue with circular deps